asm/cluster/cluster.yaml

```diff
@@ -11,13 +11,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+---
 apiVersion: container.cnrm.cloud.google.com/v1beta1
 kind: ContainerCluster
 metadata:
   annotations:
     gke.io/cluster: "bootstrap://"
-  name: "asm-cluster" # {"$ref":"#/definitions/io.k8s.cli.setters.gcloud.container.cluster"}
+  name: "asm-cluster"  # {"$ref":"#/definitions/io.k8s.cli.setters.gcloud.container.cluster"}
```

Comment:
can this extra space pass lint?

Reply:
That extra space is actually the recommended style for inline comments. (New to me too!)